client/my-sites/customer-home/style.scss

```diff
@@ -1,110 +1,174 @@
-.customer-home__layout-col {
-	@include breakpoint('>960px') {
-		width: 49%;
-	}
-}
+.customer-home {
+	&__box-action {
+		margin-bottom: 16px;
+		width: 100%;
+
+		.button {
+			border-width: 1px;
+			display: block;
+			min-height: 85px;
+			min-width: 100%;
+			padding: 24px;
+	    text-align: center;
```

Comment:
Indentation off here

Reply:
Good eye! Fixed in: https://github.com/Automattic/wp-calypso/commit/81e150b3376bee4572ded0f896b7bdf06b424fe0